lib/core/kotekanTrackers.hpp

```diff
@@ -0,0 +1,110 @@
+#ifndef KOTEKAN_TRACKERS_HPP
+#define KOTEKAN_TRACKERS_HPP
+
+#include "restServer.hpp" // for connectionInstance, restServer
+#include "visUtil.hpp"    // for StatTracker
+
+#include <map>      // for map
+#include <memory>   // for shared_ptr
+#include <mutex>    // for mutex
+#include <stddef.h> // for size_t
+#include <string>   // for string
+
+namespace kotekan {
+
+typedef std::map<std::string, std::shared_ptr<StatTracker>> stage_trackers_t;
+
+class KotekanTrackers {
```

Comment:
This is missing a class doc string/comments.   See for example `metrics` (although even that is maybe a little sparse on details).  It's good to have something that gives an overview of what the class is trying to achieve and how to use it.